packages/lib-user/src/components/shared/AllProjects/Projects/Projects.js

```diff
@@ -0,0 +1,106 @@
+import { Box, ResponsiveContext, Text } from 'grommet'
+import styled from 'styled-components'
+import { useContext } from 'react'
+import { arrayOf, bool, func, number, shape, string } from 'prop-types'
+import ProjectCard from '@zooniverse/react-components/ProjectCard'
+import Loader from '@zooniverse/react-components/Loader'
+
+import { useTranslation } from '../../../../translations/i18n.js'
+import { Pagination } from '@components/shared'
+
+const StyledBox = styled(Box)`
+  list-style: none;
+  margin-block-start: 0;
+  padding-inline-start: 0;
+  column-gap: 20px;
+  row-gap: 20px;
+`
+
+const DEFAULT_HANDLER = () => {}
+
+function Projects({
+  error,
+  loading = false,
+  numProjects = 0,
+  page = 1,
+  pageSize = 20,
+  renderedProjects = [],
+  setPage = DEFAULT_HANDLER
+}) {
+  const { t } = useTranslation()
+  const size = useContext(ResponsiveContext)
+
+  function handlePageChange({ page }) {
+    setPage(page)
+  }
```

Comment:
Will likely add some focus management here in the final iteration, but need to ask Sean first if he'd like the Pagination nav moved above the project cards for visibility.